qutebrowser/browser/webengine/webenginesettings.py

```diff
@@ -162,6 +182,30 @@ class WebEngineSettings(websettings.AbstractSettings):
         QWebEngineSettings.FantasyFont: QFont.Fantasy,
     }
 
+
+    def set_unknown_url_scheme_policy(self, policy: str) -> bool:
+        """Set the UnknownUrlSchemePolicy to use.
+
+        Return:
+            True if there was a change, False otherwise.
+        """
+        assert policy is not usertypes.UNSET  # type: ignore
+        old_value = self._settings.unknownUrlSchemePolicy()
+        policy = self._UNKNOWN_URL_SCHEME_POLICY[policy]
+        self._settings.setUnknownUrlSchemePolicy(policy)
+        return old_value != policy
+
+    def _update_setting(self, setting, value):
+        if setting == 'content.unknown_url_scheme_policy':
+            if hasattr(self, '_UNKNOWN_URL_SCHEME_POLICY'):
```

Comment:
Either having an attribute or not sounds a bit odd here - maybe set it to `None` instead and check that here? Something similar is done in other places for e.g. optional imports.